CHANGELOG.md

```diff
@@ -6,6 +6,30 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 3.4.0    
+
+#### General    
+
+* `FEAT`: add notifications ([#1505](https://github.com/camunda/camunda-modeler/issues/1505))
+* `FEAT`: add extension point for ui plugins ([#1490](https://github.com/camunda/camunda-modeler/issues/1490))
```

Comment:
Maybe we could add this feature as well:  https://github.com/camunda/camunda-modeler/commit/c1716540efde646d1b74e1827ae0a78fb02cb1ab

Maybe you can list all of these events in one section (so e.g. <app.activeTabChanged>...). This way we would get rid of the BPMN section, which only includes two of these events

Reply:
BPMN section now also contains elements fixed with bpmn-js release

Reply:
Maybe let's move this one to a single section:

```js
#### Plugins

* `FEAT`: add extension point for ui plugins ([#1490](https://github.com/camunda/camunda-modeler/issues/1490))
* Introduce several application events to hook into:
    * <tab.saved> ([#1498](https://github.com/camunda/camunda-modeler/pull/1498))
    * <tab.activeSheetChanged> ([`403afc`](https://github.com/camunda/camunda-modeler/commit/403afc920cf6c745816c3cd456baeb99830b25ed))
    * <app.activeTabChanged> ([`403afc`](https://github.com/camunda/camunda-modeler/commit/403afc920cf6c745816c3cd456baeb99830b25ed)) 
    * <bpmn.modeler.configure> ([#1499](https://github.com/camunda/camunda-modeler/issues/1499))
    * <bpmn.modeler.created> ([#1500 ](https://github.com/camunda/camunda-modeler/issues/1500))
````

Then we can remove these references  from the